.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+#### PreSubmission Checklist
```

Comment:
this does not need and issue template or  ?

Reply:
ohk... so what about PR template?

Reply:
and there will be no format when submitting PRs?

Reply:
yeah i think its fine in this case

Reply:
ok lets leave it

Reply:
ok